js/background.js

```diff
@@ -978,6 +978,30 @@
       }
     });
 
+    Whisper.events.on('onShowUserDetails', async ({ userPubKey }) => {
+      const conversation = await ConversationController.getOrCreateAndWait(
+        userPubKey,
+        'private'
+      );
+
+      const avatarPath = conversation.getAvatarPath();
+      const profile = conversation.getLokiProfile();
+      const displayName = profile && profile.displayName;
+
+      if (appView) {
+        appView.showUserDetailsDialog({
+          profileName: displayName,
+          pubkey: userPubKey,
+          avatarPath,
+          avatarColor: conversation.getColor(),
+          onOk: async () => {},
+          onStartConversation: async () => {
+            window.Whisper.events.trigger('showConversation', userPubKey);
```

Comment:
I think you can drop `window` here (`Whisper` is already declared as a global at the top of the file)
also there is no need for `async` since there is no `await` call :)